progressive-web-app/src/dx-excshell-1/web-src/src/utils.js

```diff
@@ -0,0 +1,57 @@
+/* 
+* <license header>
+*/
+
+/* global fetch */
+
+/**
+ *
+ * Invokes a web action
+ *
+ * @param  {string} actionUrl
+ * @param {object} headers
+ * @param  {object} params
+ *
+ * @returns {Promise<string|object>} the response
+ *
+ */
+
```

Comment:
nit: newline shouldn't be here

Reply:
this is in the template, and every app created with it.